lib/js/components/Form/SelectionControl/SelectionControl.vue

```diff
@@ -0,0 +1,352 @@
+<template>
+	<div
```

Comment:
most likely, if you convert this `div` into a `label` then you could remove `click` handler. Clicks on a `label` tag automagically toggles checkbox/radio

Reply:
but it is not good practice to use div in label

Reply:
why don't you use `spans` inside label?